maestroConfig/deviceDBConfigMonitor.go

```diff
@@ -307,18 +313,24 @@ func (ddbConfig *DDBConfig) Put(t interface{}) (err error) {
 		if err == nil {
 			var config ConfigWrapper
 			config.Body = string([]byte(bodyJSON))
-			config.Relay = ""
+			config.Relay = ddbConfig.ConfigClient.Relay
 			config.Name = ddbConfig.Key
+			config.Id = ""
 
 			//Marshal the storage object to put into deviceDB
-			bodyJSON, err := json.Marshal(&config)
+			payloadJSON, err := json.Marshal(&config)
 
 			//log.MaestroInfof("DDBConfig.Put(): bodyJSON: %s\n", bodyJSON)
 			if err == nil {
 				var devicedbClientBatch *client.Batch
 				ctx, _ := context.WithCancel(context.Background())
 				devicedbClientBatch = client.NewBatch()
-				devicedbClientBatch.Put(ddbConfig.Key, string([]byte(bodyJSON)), "")
+
+				//format the devicedb keyname
+				name := fmt.Sprintf("%v.%v.%v", ddbConfig.ConfigClient.Prefix, ddbConfig.ConfigClient.Relay, ddbConfig.Key)
```

Comment:
maybe this key formulation should be in a small helper function since it's used in several places.